crates/kitsune_p2p/kitsune_p2p/src/spawn/actor.rs

```diff
@@ -713,7 +713,12 @@ impl KitsuneP2pActor {
                         }
                     })
                     .await;
-                panic!("KitsuneP2p: networking poll shutdown");
+
+                tracing::error!(
+                    "KitsuneP2p: networking poll shutdown. Networking will no longer work! 
+                You can ignore this is if it happened during node shutdown. 
+                Otherwise please restart your node and report this error."
```

Comment:
Should Holochain know when it's shutting down? The process would have been signalled so it should be possible to decide here whether this loop has exited correctly or not. Not for this PR, just thinking ahead to an improvement here

Reply:
Makes sense thanks, I guess that's something we could plumb through from the kitsune 'host' at some point